libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/RestClient.swift

```diff
@@ -38,11 +38,9 @@ public enum RestClientError: Error {
 }
 
 public struct RestResponse {
-    private static let emptyJsonDictionaryArrayResponse = [String: Any]()
-    private static let emptyJsonArrayResponse = [[String: Any]]()
     private static let emptyStringResponse = ""
     private (set) var data: Data
```

Comment:
@trooper2013 , we need to the same for the data property above to expose the data with public private(set)

Reply:
there's an api already for that. asData()